scripteditor.js

```diff
@@ -133,7 +133,7 @@ function getData() {
 
 function getCoins() {
 
-  var url = 'https://api.coinmarketcap.com/v1/ticker/?convert=' + targetCurrency + '&limit=0?';
+  var url = 'https://api.coinmarketcap.com/v1/ticker/?convert=' + targetCurrency + '&limit=0';
```

Comment:
Everything looks good.  Can you change this line to the following though, per #21 ?

`var url = 'https://api.coinmarketcap.com/v1/ticker/limit=0?&convert=' + targetCurrency;`

I'll approve it after that.